bciers/apps/reporting/src/app/components/datagrid/models/operationColumns.tsx

```diff
@@ -92,8 +92,11 @@ const ActionCell = (params: GridRenderCellParams) => {
     <Button
       color="primary"
       onClick={async () => {
-        const reportingYear = await getReportingYear();
-        const newReportId = await handleStartReport(OperationId, reportingYear);
+        const reportingYearObj = await getReportingYear();
+        const newReportId = await handleStartReport(
+          OperationId,
+          reportingYearObj.reporting_year,
+        );
```

Comment:
We can leave that as a tech debt item:
2 API calls in a row to execute logic on the client side is a bit of a code smell
- Either the reporting year is pulled from the page (The operations page already fetches it for its display, we can grab that value from there),
- Or the "start report" API call grabs the current reporting year in its own server implementation